src/afl-cc.c

```diff
@@ -2091,10 +2102,11 @@ void add_native_pcguard(aflcc_state_t *aflcc) {
       "pcguard instrumentation with pc-table requires LLVM 6.0.1+"
       " otherwise the compiler will fail");
   #endif
+
   if (aflcc->instrument_opt_mode & INSTRUMENT_OPT_CODECOV) {
 
     insert_param(aflcc,
-                 "-fsanitize-coverage=trace-pc-guard,bb,no-prune,pc-table");
+                "-fsanitize-coverage=trace-pc-guard,bb,no-prune,pc-table");
```

Comment:
can you please do "make code-format"? this will remove unnecessary indent changes

Reply:
The codebase seems not formatted for a while and running `make code-format` produces lots of unrelated changes so I just formatted the code I editted.